src/site/content/en/lighthouse-performance/unused-javascript/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Remove unused JavaScript
+description: |
+  Learn how to pass Lighthouse's "Remove unused JavaScript" audit.
+web_lighthouse:
+  - unused-javascript
+date: 2020-07-07
+---
+
+Unused JavaScript can slow down your page load speed:
+
+* If the JavaScript is [render-blocking][crp], the browser must
+  download, parse, compile, and evaluate the script before it can proceed
+  with all of the other work that's needed for rendering the page.
+* Even if the JavaScript is asynchronous (i.e. not render-blocking), the
+  code competes for bandwidth with other resources while it's downloading,
+  which has significant performance implications. Sending unused code over
+  the network is also wasteful for mobile users who don't have unlimited
+  data plans.
```

Comment:
@connorjclark check out this updated intro

Reply:
looks cool, thanks!